request.cc

```diff
@@ -0,0 +1,120 @@
+#include "request.h"
+
+#include <boost/tokenizer.hpp>
+#include <boost/algorithm/string.hpp>
+
+using Headers = std::vector<std::pair<std::string, std::string>>;
+
+// Helper function to tokenize a string
+boost::tokenizer<boost::char_separator<char>> 
+token_generator(const std::string s, const char* sep) {
```

Comment:
Could we rename this helper to follow our method naming convention of `camelCase()`?